thinc/loss.py

```diff
@@ -225,7 +275,7 @@ def __call__(
         self, guesses: Sequence[Floats2d], truths: Sequence[IntsOrFloatsOrStrs]
     ) -> Tuple[List[Floats2d], float]:
         grads = self.get_grad(guesses, truths)
-        loss = self._get_loss_from_grad(grads)
+        loss = self.get_loss(guesses, truths)
```

Comment:
Do we want `_get_loss` here? I think the input validation is already done through `get_grad`?

Reply:
Oh, nevermind.